tests/analyze/test_analyze_shared.py

```diff
@@ -42,38 +42,42 @@ class Askalono_result:
 class AnalyzeSharedTestCase(unittest.TestCase):
     def test_add_analyze_to_dep_tree_simple(self):
         tree = AnyNode(
-            id="root",
+            name="root",
+            version="1.2.3",
             children=[
-                AnyNode(id="child", children=[AnyNode(id="childChild")]),
-                AnyNode(id="child2", children=[AnyNode(id="child2Child")]),
+                AnyNode(name="child", version="1.0.0", children=[AnyNode(name="childChild", version="1.0.0")]),
+                AnyNode(name="child2", version="1.0.1", children=[AnyNode(name="child2Child", version="9.9.9")]),
             ],
         )
+        root_id = get_node_id("root", "1.2.3")
+        childChild_Id = get_node_id("childChild", "1.0.0")
         analyze = {
-            "root": [{"data": "Hey! I am a license text", "name": "Awesome license"}],
-            "childChild": [
+            root_id: [{"data": "Hey! I am a license text", "name": "Awesome license"}],
+            childChild_Id: [
```

Comment:
`child_child_id`

Reply:
well technically childChild is a name not a par of variable but ok :D

Reply:
@S4n60w3n you're right, but it's a bit disturbing, :) but in general you're right, we can keep this